tests/runtime/variants/host.ts

```diff
@@ -36,6 +36,10 @@ async function run() {
   wasm.testImports();
   assert.deepStrictEqual(wasm.roundtripOption(1), 1);
   assert.deepStrictEqual(wasm.roundtripOption(null), null);
+  // @ts-ignore
```

Comment:
For now I've just ignored this behaviour from a typescript perspective - indicating that function parameters are optional to TypeScript would be a nice thing to add but requires `isnullable` information at the param writer level.

Reply:
https://github.com/bytecodealliance/wit-bindgen/issues/404